bootstrap-config/data-workspace-apps-copilot.yml

```diff
@@ -113,26 +73,8 @@ services:
       url: data-store-service-staging.london.cloudapps.digital
   image_location: public.ecr.aws/uktrade/copilot-bootstrap:latest
   name: data-store-service
-  overlapping_secrets:
-  - FLASK_ENV
   repo: git@github.com:uktrade/data-store-service.git
-  secrets:
-    AUTHBROKER_CLIENT_ID: AUTHBROKER_CLIENT_ID
-    AUTHBROKER_CLIENT_SECRET: AUTHBROKER_CLIENT_SECRET
-    AUTHBROKER_URL: AUTHBROKER_URL
-    AWS_ACCESS_KEY_ID: AWS_ACCESS_KEY_ID
-    AWS_DEFAULT_REGION: AWS_DEFAULT_REGION
-    AWS_SECRET_ACCESS_KEY: AWS_SECRET_ACCESS_KEY
-    CSV_SAMPLE_INFER_LINES: CSV_SAMPLE_INFER_LINES
-    DATAFLOW_BASE_URL: DATAFLOW_BASE_URL
-    DATAFLOW_HAWK_ID: DATAFLOW_HAWK_ID
-    DATAFLOW_HAWK_KEY: DATAFLOW_HAWK_KEY
-    DSS_PAGINATION_SIZE: DSS_PAGINATION_SIZE
-    FLASK_ENV: FLASK_ENV
-    INPUTS_SOURCE_FOLDER: INPUTS_SOURCE_FOLDER
-    S3_BUCKET_URL: S3_BUCKET_URL
-    SENTRY_DSN: SENTRY_DSN
-    SENTRY_ENVIRONMENT: SENTRY_ENVIRONMENT
+  secrets: {}
```

Comment:
Odd 🤔

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/17/commits/8d8a41c3b591670596c29d2cd72ee4a66b4dffec.